synthtool/languages/java.py

```diff
@@ -106,9 +126,13 @@ def fix_proto_headers(proto_root: Path) -> None:
     )
 
 
-def fix_grpc_headers(grpc_root: Path, package_name: str) -> None:
+def fix_grpc_headers(grpc_root: Path, package_name: str = "unused") -> None:
```

Comment:
Please add a doc comment string for this function.

Reply:
Done